auth/ipa_ad_trust.adoc

```diff
@@ -0,0 +1,104 @@
+== Requirements
+
+The following is needed in order to enable AD Trust Authentication to the Appliance:
+
+* A CentOS/RHEL 7.2 based MiQ Appliance
+* Windows Server 2008 R2 or later with configured AD DC and DNS installed locally on the DC
+
+[[configure-ipa-ad-trust-server]]
+== Setup and configure an IPA server for AD Trust Authentication
+
+* IPA Server based on FreeIPA 3.3.3 or later is recommended
+
+Instructions for setting up and configure cross-realm trust between an IPA domain and an AD (Active Directory) domain
+is described on http://www.freeipa.org/page/Active_Directory_trust_setup[freeipa.org Active Directory Trust Setup]
+
+Special care should be made when configuring DNS as improper DNS configurations can result in poor performance
+and improper functionality.  For more details refer to the following documents:
+
+* http://www.freeipa.org/page/Active_Directory_trust_setup#DNS_configuration[freeipa.org Active Directory Trust Setup/DNS configuration]
+* http://www.freeipa.org/page/DNS[freeipa.org DNS]
+* https://www.freeipa.org/page/Howto/DNSSEC[freeipa.org DNSSEC]
+
+[[enable-external-auth]]
+== Enable the MiQ Appliance to use the configued IPA server
+
+Using the Appliance Console, enable external authentication to the IPA server.
+
+1. Login to console as _root_
+2. Run _appliance_console_
+3. Summary screen should show *External Auth* as _not configured_, Press any key
+4. From the Advanced Setting menu, select the menu item *Configure External Authentication (httpd)*
+5. Enter the FQDN of the IPA Server, i.e. _ipaserver.test.company.com_
+6. Enter the IPA Server domain, i.e. _test.company.com_
+7. Enter the IPA Server realm, i.e. _TEST.COMPANY.COM_
+8. Press enter to select the default IPA Server Principal, i.e. _admin_
+9. Enter the Password of the IPA Server Principal
+10. Review details, and Enter *y* to proceed.
+
+[[configuring-admin-ui]]
+== Configuring the MiQ appliance to use external authentication
+
+Login to the MiQ appliance as admin, then in _Settings->Configuration->Server->Authentication_
+
+* Set mode to External (httpd)
+* Check: _Get User Groups from External Authentication (httpd)_
+* _Optionally_ Check: _Enable Single Signon_
+* Click Save.
+
+The above steps need to be done on each UI and WebService enabled appliance.
+
+in _Settings->Configuration->Access Control_
+
+* Make sure the user's groups are created on the Appliance and appropriate roles assigned to those groups.
+
+[[create-group-ui]]
+== Create groups on the MiQ appliance
+
+Login to the MiQ appliance as admin, then in _Settings->Configuration->Access Control->Groups->Configuration->Add a new Group_
+
+* Check: _(Look Up LDAP Groups)_
+* Enter the AD user as the _User to Look Up_ i.e. _ipauser@ipaserver.test.company.com_
+* Click Retrieve.
+* Choose a group from the _LDAP Groups for User_ dropdown.
+* Assign the appropriate roles to the group
+* Click Add.
+
+The above steps need to be done on each UI and WebService enabled appliance.
```

Comment:
You should probably move this ABOVE the steps so that they know before they have read it where to perform these steps.